nirodents/workflows/brainextraction.py

```diff
@@ -156,14 +146,11 @@ def init_rodent_brain_extraction_wf(
     wf = pe.Workflow(name)
 
     # truncate target intensity for N4 correction
-    clip_target = pe.Node(niu.Function(function=_trunc), name="clip_target")
-    clip_target.inputs.percentiles = (None, 99.9)
-    clip_target.inputs.clip_max = None
+    clip_target = pe.Node(IntensityClip(p_min=0, p_max=99.9), name="clip_target")
 
     # truncate template intensity to match target
-    clip_tmpl = pe.Node(niu.Function(function=_trunc), name="clip_tmpl")
+    clip_tmpl = pe.Node(IntensityClip(p_min=35, p_max=90), name="clip_tmpl")
```

Comment:
```suggestion
    clip_tmpl = pe.Node(IntensityClip(p_min=5, p_max=98), name="clip_tmpl")
```

Reply:
These values worked well with the template though